src/Compilers/VisualBasic/Portable/Symbols/Metadata/PE/PENamedTypeSymbol.vb

```diff
@@ -1274,6 +1274,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols.Metadata.PE
         End Function
 
         Private Function CalculateUseSiteInfoImpl() As UseSiteInfo(Of AssemblySymbol)
+            ' GetCompilerFeatureRequiredDiagnostic depends on this being the highest priority use-site diagnostic. If another
```

Comment:
>' GetCompilerFeatureRequiredDiagnostic depends on this being the highest priority use-site diagnostic. If another [](http://example.com/codeflow?start=12&length=113)

It looks like ```_lazyCachedUseSiteInfo``` can be initialized during object construction and we will never get here.

Reply:
Looking at the implementation, it seems like the only time that will happen is if there was an error reading basic method definition flags from the assembly. I think that's probably fine, as it will always be a BindToBogus. I'll update the comment and assert.